Explorer/Assets/DCL/SDKComponents/Tween/Systems/TweenUpdaterSystem.cs

```diff
@@ -17,19 +17,24 @@
 using ECS.Unity.Groups;
 using ECS.Unity.Transforms.Components;
 using System.Collections.Generic;
+using CrdtEcsBridge.Components.Transform;
+using ECS.Groups;
+using ECS.Unity.Transforms.Systems;
 using UnityEngine;
 using static DCL.ECSComponents.EasingFunction;
 using static DG.Tweening.Ease;
 
 namespace DCL.SDKComponents.Tween.Systems
 {
-    [UpdateInGroup(typeof(ComponentInstantiationGroup))]
-    [UpdateAfter(typeof(TweenLoaderSystem))]
+    [UpdateInGroup(typeof(SyncedSimulationSystemGroup))]
+    [UpdateBefore(typeof(UpdateTransformSystem))]
     [LogCategory(ReportCategory.TWEEN)]
     [ThrottlingEnabled]
     public partial class TweenUpdaterSystem : BaseUnityLoopSystem, IFinalizeWorldSystem
```

Comment:
Can you clean-up a bit (and clarify your intentions) what you pass by `ref` and by value-type copy here and `TweenSDKComponentHelper`:
- `Entity` and `CRDTEntity` should be always passed by copy (without any modifiers) as they are super-lightweight and it's cheaper to copy them than preserve a reference
- You should not pass by `ref` arguments you have no intention to modify the original reference of, consider passing them by `in` instead. For classes you can fully omit the modifier